tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source.ts

```diff
@@ -59,12 +65,34 @@ export class QueryParamsFeatureFlagDataSource extends TBFeatureFlagDataSource {
       featureFlags.enabledColorGroup =
         params.get(ENABLE_COLOR_GROUP_QUERY_PARAM_KEY) !== 'false';
     }
+
+    if (params.has(ENABLE_DARK_MODE_QUERY_PARAM_KEY)) {
+      featureFlags.enableDarkMode =
+        params.get(ENABLE_DARK_MODE_QUERY_PARAM_KEY) !== 'false';
+    }
+
     return featureFlags;
   }
 
   protected getParams() {
     return util.getParams();
   }
+
+  protected getPartialFeaturesFromMediaQuery(): {enableDarkMode?: boolean} {
+    // Set feature flag value for query parameters that are explicitly
+    // specified. Feature flags for unspecified query parameters remain unset so
+    // their values in the underlying state are not inadvertently changed.
```

Comment:
Let's update this comment still refers to "query parameters" although the code below reads media queries.  Optionally, I think the comment could also be dropped, given the comment below on L88